tools/ci.psm1

```diff
@@ -309,7 +319,11 @@ function Invoke-CITest
             ExcludeTag = $ExcludeTag
         }
 
-        Start-PSPester @arguments -Title "Pester Elevated - $TagSet"
+        $title = "Pester Elevated - $TagSet"
```

Comment:
i suppose the above could be done in the `$arguments` declaration as:
`Title = $TitlePrefix ? "$TitlePrefix - $title" : $title`
but this is fine.

Reply:
Some of this code run in Windows PowerShell, so I cannot use that operator.